java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -181,4 +196,48 @@ public CompletableFuture<String> set(
         String[] arguments = ArrayUtils.addAll(new String[] {key, value}, options.toArgs());
         return commandManager.submitNewCommand(SetString, arguments, this::handleStringOrNullResponse);
     }
+
+    @Override
+    public CompletableFuture<Long> zadd(
+            @NonNull String key,
+            @NonNull Map<String, Double> membersScoresMap,
+            @NonNull ZaddOptions options,
+            boolean changed) {
+        String[] changedArg = changed ? new String[] {"CH"} : new String[] {};
+
+        String[] membersScores =
+                membersScoresMap.entrySet().stream()
+                        .flatMap(e -> Stream.of(e.getValue().toString(), e.getKey()))
+                        .toArray(String[]::new);
+
+        String[] arguments =
+                Stream.of(new String[] {key}, options.toArgs(), changedArg, membersScores)
+                        .flatMap(Stream::of)
+                        .toArray(String[]::new);
+        return commandManager.submitNewCommand(Zadd, arguments, this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<Double> zaddIncr(
+            @NonNull String key, @NonNull String member, double increment, @NonNull ZaddOptions options) {
+        String[] arguments =
+                Stream.of(
+                                new String[] {key},
+                                options.toArgs(),
+                                new String[] {"INCR", Double.toString(increment), member})
+                        .flatMap(Stream::of)
+                        .toArray(String[]::new);
```

Comment:
You can ArrayUtils.addAll to do the same thing: 
```
String[] arguments = ArrayUtils.addAll(new String[] {key}, options.toArgs(), new String[] {"INCR", Double.toString(increment), member});
```

Reply:
```
error: incompatible types: inference variable T has incompatible bounds
        String[] arguments = ArrayUtils.addAll(new String[] {key}, options.toArgs(), new String[] {"INCR", Double.toString(increment), member});
                                              ^
    lower bounds: String,Object
    lower bounds: String[],String
  where T is a type-variable:
    T extends Object declared in method <T>addAll(T[],T...)
```